### DISCUSSION TITLE
23.11.17 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#69] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
- 내일은 어떤 일을 하나요?
    - 우재님 코드 리뷰
    - 친구 관계 테스트 코드 작성

**Comment by Jeonwoojae** [comment]
- 내일은 어떤 일을 하나요?
  - 도메인 병합
  - 민주님 코드 리뷰

**Comment by Nine-JH** [comment]
* 내일은 어떤 일을 하나요?
  * 코드리뷰
  * PR
